packages/stencil-store/src/observable-map.ts

```diff
@@ -38,49 +39,49 @@ export const createObservableMap = <T extends { [key: string]: any }>(
     },
   });
 
-  const subscribe = (subscription: StoreSubscription<T> | StoreSubscriptionObject<T>): void => {
-    subscriptions.push(toSubscription(subscription));
+  const on: OnHandler<T> = (eventName, callback) => {
+    let listeners: any[] = setListeners;
```

Comment:
Can we throw if `eventName` is not a valid one?

Reply:
ok